pkg/plugin/infiniband/filesystem_test.go

```diff
@@ -2,7 +2,7 @@ package infiniband
 
 import "testing/fstest"
 
-var embeddedFs = fstest.MapFS{ // nolint unused
+var testFS = fstest.MapFS{ // nolint unused
 	"infiniband/mlx5_ib0/ports/1/counters/excessive_buffer_overrun_errors": &fstest.MapFile{
```

Comment:
nit: don't know if this affects anything but `mlx5_ib` should now be `mlx` right?

Reply:
`mlx5_ib` in the test data is fine, since the actual matcher is on the `mlx` prefix